tests/schedule/test_regular_schedule_strategy.py

```diff
@@ -22,6 +22,18 @@ def test_not_spawning_before_start_tick(
         strategy = request.getfixturevalue(strategy)
         assert not strategy.should_spawn(tick=tick)
 
+    @pytest.mark.parametrize(
+        "strategy,tick", (("regular_strategy", tick) for tick in [20000])
+    )
+    def test_not_spawning_after_end_tick(
+        self,
+        strategy: RegularScheduleStrategy,
+        tick: int,
+        request: pytest.FixtureRequest,
+    ):
+        strategy = request.getfixturevalue(strategy)
+        assert not strategy.should_spawn(tick=tick)
```

Comment:
```suggestion
    @pytest.mark.parametrize(
        "tick", [20000]
    )
    def test_not_spawning_after_end_tick(
        self,
        regular_strategy: RegularScheduleStrategy,
        tick: int
    ):
        assert not regular_strategy.should_spawn(tick=tick)
```
You could change this test (and those like it) to use the fixture directly. I think this would make it easier to understand, because the parametrizing has only one parameter and you don't need to use FixtureRequest anymore.

Reply:
Good idea. Thanks :)